util/src/main/java/tc/oc/pgm/util/bukkit/Sounds.java

```diff
@@ -0,0 +1,69 @@
+package tc.oc.pgm.util.bukkit;
+
+import static tc.oc.pgm.util.bukkit.MiscUtils.MISC_UTILS;
+
+import net.kyori.adventure.sound.Sound;
+
+public interface Sounds {
+  Sound ADMIN_CHAT = sound("ORB_PICKUP", "ENTITY_EXPERIENCE_ORB_PICKUP", 1f, 0.7f);
+  Sound ALERT = sound("NOTE_PLING", "BLOCK_NOTE_BLOCK_PLING", 1f, 2f);
+  Sound CONTROL_POINT_GOOD = sound("PORTAL_TRAVEL", "BLOCK_PORTAL_TRAVEL", 0.35f, 2f);
+  Sound CONTROL_POINT_BAD = sound("BLAZE_DEATH", "ENTITY_BLAZE_DEATH", 0.4f, 0.8f);
+  Sound DEATH_OWN =
+      sound("IRONGOLEM_DEATH", "ENTITY_IRON_GOLEM_DEATH"); // Own death is normal pitch, full volume
+  Sound DEATH_KILLED = sound(
+      "IRONGOLEM_DEATH",
+      "ENTITY_IRON_GOLEM_DEATH",
+      0.75f,
+      4f / 3f); // Kill is higher pitch, quieter
```

Comment:
Move the comment(s) above the sound itself to try and avoid them extending several lines:
```suggestion
  // Kill is higher pitch, quieter
  Sound DEATH_KILLED = sound("IRONGOLEM_DEATH", "ENTITY_IRON_GOLEM_DEATH", 0.75f, 4f / 3f);
```